app/src/main/java/woowacourse/payments/ui/PaymentCardsScreen.kt

```diff
@@ -0,0 +1,70 @@
+@file:Suppress("DEPRECATION")
+
+package woowacourse.payments.ui
+
+import android.app.Activity
+import android.widget.Toast
+import androidx.activity.compose.rememberLauncherForActivityResult
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import woowacourse.payments.R
+import woowacourse.payments.ui.component.PaymentCardsContent
+import woowacourse.payments.ui.component.PaymentCardsTopBar
+import woowacourse.payments.ui.model.EXTRA_CARD
+import woowacourse.payments.ui.model.PaymentCardUiModel
+import woowacourse.payments.ui.theme.AndroidpaymentsTheme
+
+@Composable
+fun PaymentCardsScreen() {
+    val context = LocalContext.current
+    val toastMessage = stringResource(R.string.toast_card_add)
+    var paymentCards by rememberSaveable { mutableStateOf(listOf<PaymentCardUiModel>()) }
+
+    val cardAddLauncher =
+        rememberLauncherForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
+            if (result.resultCode == Activity.RESULT_OK) {
+                val newPaymentCard = result.data?.getParcelableExtra<PaymentCardUiModel>(EXTRA_CARD)
+                if (newPaymentCard != null) {
+                    paymentCards = paymentCards + newPaymentCard
+                    Toast.makeText(context, toastMessage, Toast.LENGTH_SHORT).show()
```

Comment:
```suggestion
                    Toast.makeText(context, context.getString(R.string.toast_card_add), Toast.LENGTH_SHORT).show()
```
위에서 받아온 context를 쓰지 않고 toastMessage라고 따로 빼둔 이유가 무엇인가요?🤔

Reply:
stringResource가 Composable 함수이기 때문에 리컴포지션 될 때마다 현재 Locale이나 Configuration change를 반영해서 최신 문자열을 가져옵니다.

예를 들어서 앱 실행 도중에 시스템 언어를 바꾸면, 토스트 메시지 값도 리컴포지션 시점에 새 문자열로 업데이트가 되기 때문에 toastMessage라고 따로 상태처럼 기억해두고 쓰도록 했습니다!

하지만 이번에 제리의 리뷰를 참고해서 해당 로직들을 전부 Activity로 옮기며 토스트 메시지 자체가 리컴포지션과 무관해졌기 때문에 다음처럼 수정했습니다!

[refactor: 카드 추가 로직을 Activity로 이동](https://github.com/woowacourse/android-payments/pull/38/commits/cb36e671500aa68e3c7aab4d4751c8fe27d9ad4e)